spec/unit/ec2_server_create_spec.rb

```diff
@@ -360,6 +360,23 @@
       end
     end
 
+    context "command-line provided secret file" do
+      before(:each) do
+        Chef::Config[:knife][:secret_file] = "sys-knife-secret-file"
+      end
+
+      it "prefers using a provided value instead of the knife confiuration" do
+        subject.config[:secret_file] = "cli-provided-secret-file"
+        bootstrap.config[:encrypted_data_bag_secret_file] = 
```

Comment:
Assignment should not be there. It's not verifying the code that needs to be tested.